src/periphery/MgvReader.sol

```diff
@@ -87,6 +88,57 @@ contract MgvReader {
     MGV = IMangrove(payable(mgv));
   }
 
+  // # Config view functions
+
+  /* Returns the configuration in an ABI-compatible struct. Should not be called internally, would be a huge memory copying waste. Use `config` instead. */
+  function configInfo(OLKey memory olKey)
+    external
+    view
+    returns (MgvStructs.GlobalUnpacked memory _global, MgvStructs.LocalUnpacked memory _local)
+  {
+    unchecked {
+      (MgvStructs.GlobalPacked __global, MgvStructs.LocalPacked __local) = MGV.config(olKey);
+      _global = __global.to_struct();
+      _local = __local.to_struct();
+    }
+  }
+
+  /* Sugar for getting only global/local config */
+  function local(OLKey memory olKey) public view returns (MgvStructs.LocalPacked) {
+    (, MgvStructs.LocalPacked _local) = MGV.config(olKey);
+    return _local;
+  }
```

Comment:
Since the access to packed global is in MgvView it increases surprise to have local here, is it OK to move it to MgvView?

Reply:
Agreed, will do.

I tried to minimize `MgvView`, but this one felt weird to when I moved it back to `MgvReader`, which is a sign that you're right :-)

Reply:
Addressed in a24822f19c4b277facd56a3d25a6690a03723944